test/EFCore.PG.FunctionalTests/TestUtilities/TestEnvironment.cs

```diff
@@ -39,5 +39,23 @@ public static Version PostgresVersion
                 return _postgresVersion = conn.PostgreSqlVersion;
             }
         }
+
+        private static bool? _postgisAvailable;
+
+        public static bool PostgisAvailable
+        {
+            get
+            {
+                if (_postgisAvailable.HasValue)
+                    return _postgisAvailable.Value;
+                using var conn = new NpgsqlConnection(NpgsqlTestStore.CreateConnectionString("postgres"));
+                conn.Open();
+                using var cmd = conn.CreateCommand();
+
+                cmd.CommandText = "SELECT 1 FROM pg_available_extensions WHERE \"name\" = 'postgis' LIMIT 1";
+                _postgisAvailable = cmd.ExecuteNonQuery() > 0;
```

Comment:
Uh, I don't think this works - ExecuteNonQuery always returns -1 for SELECT as we recently discussed, no? You can just do `SELECT EXISTS(SELECT * FROM pg_available_extensions WHERE name = 'intagg')` and use ExecuteScalar to read it directly as a bool.

Reply:
I'm dumb, sorry 😢